src/components/button/button.md

```diff
@@ -0,0 +1,11 @@
+```jsx
+<Button>Default Button</Button>
+```
+
+```jsx
+<Button color="primary">Primary Button</Button>
+```
+
+```jsx
+<Button color="secondary">Secondary Button</Button>
```

Comment:
Might I recommend that we consider a blank button too? I think we use that a lot and it might be a good one for public consumption. Really a product / design questions so consider this just a "let's think about it" ;)

Reply:
I'm using the requirements of the Maven Page for this. 

So you're in luck because it *does* use a cancel button that looks like a link. I am actually going to remove secondary for now and only include Primary and, uh, Blank.

Reply:
Makes sense and I think we realized that during the preview the other day 👍